spec/lib/miq_automation_engine/service_methods/miq_ae_service_miq_user_role_spec.rb

```diff
@@ -0,0 +1,12 @@
+module MiqAeServiceMiqUserRoleSpec
+  describe MiqAeMethodService::MiqAeServiceMiqUserRole do
+    before(:each) do
```

Comment:
@jockey10 
you can remove the (:each) since that is default
```
before do
  @role = FactoryGirl.create(:miq_user_role, :name => "Role1")
end
```

A better approach would be to use let syntax
```
let(:role) { FactoryGirl.create(:miq_user_role, :name => 'Role1')  }
```

then you don't need the before block. When you refer to role anywhere in your spec the role would get defined.